xsuite/build_wasms.sh

```diff
@@ -3,19 +3,22 @@ TARGET_DIR="$(pwd)/../target"
 SOURCE_DIR="$(pwd)/contracts"
 DEST_DIR="/tmp/xsuite-contracts"
 
+rm -rf $TARGET_DIR
 rm -rf $DEST_DIR
 cp -R $SOURCE_DIR $DEST_DIR
 
-pnpm xsuite build --dir $DEST_DIR -r --locked --target-dir $TARGET_DIR
+pnpm xsuite build-reproducible $DEST_DIR -r --image multiversx/sdk-rust-contract-builder:v8.0.0 --output-dir $TARGET_DIR
 
-cd $DEST_DIR
+cd $TARGET_DIR
 
 shopt -s globstar dotglob
```

Comment:
I also don't think we need anymore this line and all what is below?

Basically now we could just commit the .wasm inside output-reproducible and make the tests use them.

At the end, I'm not even sure we need this file anymore? Which proves the utility of your work!